README.md

```diff
@@ -170,6 +170,9 @@ Cloud Run is on [Stackshare](https://stackshare.io/google-cloud-run) and [StackO
 
 * 📰+📦 [Step by step of many Cloud Run concepts](https://github.com/meteatamel/cloudrun-tutorial)
 * 🎬+📦 [Deploying a CI/CD Nuxt.js site in universal mode with Google Cloud Run and Cloud Build](https://youtu.be/WRIiZ6KkYL0)
+* ✏️+📦 [Qwiklabs Course - Application Development with Cloud Run](https://www.qwiklabs.com/course_templates/371)
```

Comment:
Thanks for the PR. 
I already have added the Qwiklabs today under "Interactive tutorials" at the top (I assumed it was interactive)

Maybe we can start a new section "Courses" and move these three under it?

Reply:
Good idea. 
I opened up a new Courses section and added them there